bindata/assets/kube-apiserver/pod.yaml

```diff
@@ -102,7 +102,7 @@ spec:
       httpGet:
         scheme: HTTPS
         port: 6443
-        path: livez
+        path: livez?verbose&exclude=etcd
```

Comment:
We should also start scraping https://kubernetes.io/docs/reference/instrumentation/slis/ so that we could plot how good or bad our healthchecks actually are.

Reply:
/cc @dgrisonnet